physx/include/foundation/PxPreprocessor.h

```diff
@@ -196,7 +196,7 @@ define anything not defined through the command line to 0
 	#define PX_DEBUG 0
 #endif
 #ifndef PX_CHECKED
-	#define PX_CHECKED 0
+	#define PX_CHECKED 1
```

Comment:
Is there a reason you changed `PX_CHECKED` to 1? I think in default it should stay at 0.